marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/entities/search/addPropertiesToSearchResponse.sjs

```diff
@@ -373,8 +373,54 @@ function verifyResultsWithoutSelectedProperties() {
   test.assertEqual(7, response.entityPropertyDefinitions.length);
 }
 
+function verifyPrimaryKeyWithDefinedEntities() {
+  const response = {
+    "snippet-format": "snippet",
+    "total": 2,
+    "results": [
+      {
+        "index": 1,
+        "uri": "/content/jane.json",
+      },
+      {
+        "index": 2,
+        "uri": "/content/sally.json",
+      }
+    ]
+  };
+  entitySearchLib.addPropertiesToSearchResponse(entityName, response);
+  test.assertEqual(101, response.results[0].primaryKey.propertyValue);
```

Comment:
@saarimrahman @rahulvudutala All of the functions in here need to return an array of assertions so that they all get reported back (as opposed to JUnit, where you only have to call test functions). 

So this should be:

```
  return [
    test.assertEqual(101, response.results[0].primaryKey.propertyValue),
    test.assertEqual("customerId", response.results[0].primaryKey.propertyPath),
    test.assertEqual(101, response.results[1].primaryKey.propertyValue),
    test.assertEqual("customerId", response.results[1].primaryKey.propertyPath)
  ]
```

That needs to be done for all of the functions.